database/buildSchema/41_application_list_view.sql

```diff
@@ -42,13 +44,15 @@ CREATE FUNCTION application_list (userid int DEFAULT 0)
         reviewers,
         reviewer_action,
         assigner_action,
-        CASE WHEN is_fully_assigned_level_1 IS NULL THEN
-            FALSE
-        ELSE
-            is_fully_assigned_level_1
-        END,
-        assigned_questions_level_1,
-        total_questions
+        -- CASE WHEN is_fully_assigned_level_1 IS NULL THEN
+        --     FALSE
+        -- ELSE
+        --     is_fully_assigned_level_1
+        -- END,
+        -- assigned_questions_level_1,
```

Comment:
Is this meant to be commented? Why don't we need it? (And can you get rid of it if we don't)?

Reply:
The reason I didn't remove this was that I wasn't sure if we will use this at some point. So would rather keep it, in case we might need to use this, and if not we can remove later.